zaas-service/src/test/java/org/zowe/apiml/acceptance/ZaasTest.java

```diff
@@ -49,6 +49,27 @@ class ZaasTest {
     @Value("${apiml.service.hostname:localhost}")
     private String hostname;
 
+    private final String BODY = """
+        {
+          "resourceClass": "ZOWE",
+          "resourceName": "APIML.SERVICES",
+          "accessLevel": "READ"
+        }
+        """;
+
+    @Value("${server.ssl.keyPassword}")
+    char[] password;
+    @Value("${server.ssl.keyStore}")
+    String client_cert_keystore;
+    @Value("${server.ssl.keyStore}")
+    String keystore;
```

Comment:
I suppose these can be private
```suggestion
    @Value("${server.ssl.keyPassword}")
    private char[] password;
    @Value("${server.ssl.keyStore}")
    private String client_cert_keystore;
    @Value("${server.ssl.keyStore}")
    private String keystore;
```